crates/pathfinder/examples/migrate_db.rs

```diff
@@ -32,27 +32,4 @@ fn main() {
         migrated_at - started_at,
         size_after_migration - size_before
     );
-
-    // in general one does not want to do the full vacuum because it's going to take a long time
-    if false {
-        let conn = storage.connection().unwrap();
-
-        let vacuum_started = std::time::Instant::now();
-        let vacuum_ret = conn.execute("VACUUM", []).expect("vacuum failed");
-        drop(conn);
-        drop(storage);
-
-        let vacuumed_at = std::time::Instant::now();
-
-        let size_after_vacuum = std::fs::metadata(&path)
-            .expect("Vacuuming removed the database?")
-            .len() as i64;
-
-        println!(
-            "vacuumed in {:?}, size change: {}, VACUUM returned {}",
-            vacuumed_at - vacuum_started,
-            size_after_vacuum - size_after_migration,
-            vacuum_ret
-        );
-    }
```

Comment:
I don't think it's important to remove `if false {}` "non-production" code but perhaps this wont be needed in a while, until we find a reason to transition every database to incremental vacuum...

Reply:
I'm happy to revert this?